setup/plugins/init/StandaloneUsers.civi-setup.php

```diff
@@ -49,13 +49,50 @@
           'name' => 'everyone',
           'label' => ts('Everyone, including anonymous users'),
           // Provide default open permissions
-          'permissions' => ['CiviMail subscribe/unsubscribe pages', 'make online contributions', 'view event info', 'register for events'],
+          'permissions' => ['CiviMail subscribe/unsubscribe pages', 'make online contributions', 'view event info', 'register for events', 'authenticate with password'],
         ],
         [
           'name' => 'admin',
           'label' => ts('Administrator'),
+          // @todo Would it be better to grant super user powers via
+          // 'all CiviCRM permissions and ACLs' or 'administer CiviCRM'?
```

Comment:
@artfulrobot Yeah, I think for `admin`, it makes sense to grant some wildcard permission.

Aside: For `everyone` and `staff_user`, I feel like there's a sort of low-grade pain-point on the horizon -- where extensions define permissions, and it's a bit annoying to reconfigure roles as you enable/disable extensions.  But I'm not certain what to do about that. It probably ties into the questions on dev/core#4466.